lib/tapioca/generators/gem.rb

```diff
@@ -144,7 +144,9 @@ def compile_gem_rbi(gem)
         say("Compiling #{gem_name}, this may take a few seconds... ")
 
         strictness = @typed_overrides[gem.name] || "true"
-        rbi_body_content = compiler.compile(gem, 0, @doc)
+        rbi = RBI::File.new(strictness: strictness)
```

Comment:
Aren't we going to add the strictness twice if it's also done by `rbi_header`?

Reply:
Don't you get the same result if you do not pass any strictness to the `RBI::File`?

```suggestion
        rbi = RBI::File.new
```